src/app/code/community/Alma/Installments/sql/alma_installments_setup/mysql4-install-2.0.0.php

```diff
@@ -0,0 +1,19 @@
+<?php
+$installer = $this;
+$installer->startSetup();
+$pathToDelete = [
+    'payment/alma_installments/p2x_enabled',
+    'payment/alma_installments/p2x_min_amount',
+    'payment/alma_installments/p2x_max_amount',
+    'payment/alma_installments/p3x_enabled',
+    'payment/alma_installments/p3x_min_amount',
+    'payment/alma_installments/p3x_max_amount',
+    'payment/alma_installments/p4x_enabled',
+    'payment/alma_installments/p4x_min_amount',
+    'payment/alma_installments/p4x_max_amount',
+];
+$table = $installer->getConnection()->getTableName("core_config_data");
+foreach ($pathToDelete as $path) {
+$installer->getConnection()->delete($table,array('path = ?' => $path));
```

Comment:
are we just remove them or should we do also copy configuration to new `paths` ?

Reply:
For the moment : just remove